lib/sedna/core/federated_learning/federated_learning.py

```diff
@@ -13,17 +13,23 @@
 # limitations under the License.
 
 
+import asyncio
+import sys
 import time
 
-from sedna.core.base import JobBase
-from sedna.common.config import Context
-from sedna.common.file_ops import FileOps
+from plato.clients import registry as client_registry
+from plato.config import Config
+
+from sedna.algorithms.transmitter import S3Transmitter, WSTransmitter
 from sedna.common.class_factory import ClassFactory, ClassType
-from sedna.service.client import AggregationClient
+from sedna.common.config import BaseConfig, Context
 from sedna.common.constant import K8sResourceKindStatus
+from sedna.common.file_ops import FileOps
+from sedna.core.base import JobBase
+from sedna.service.client import AggregationClient
 
 
-class FederatedLearning(JobBase):
+class FederatedLearningV0(JobBase):
```

Comment:
Remove this section if the new method is forward compatible

Reply:
updated, currently we use two interfaces and we will fix it in the next PR.